rfcs/detecting_sensitive_textual_content/20230309-implementation_plan_sensitive_terms_list.md

```diff
@@ -0,0 +1,158 @@
+
+3.  On what basis will Openverse maintainers use their "discretion" to create
+    this list?
+
+    Until our community has developed its own standards around sensitivity, we
+    can use a few existing documents in the WordPress and user-submitted content
+    ecosystem to guide us:
+
+    - WordPress's page on [etiquette](https://wordpress.org/about/etiquette/) in
+      the project.
+    - WordPress's
+      [code of conduct](https://make.wordpress.org/community/handbook/wordcamp-organizer/planning-details/code-of-conduct/)
+      for in-person events.
+    - YouTube's
+      [community guidelines](https://www.youtube.com/howyoutubeworks/policies/community-guidelines/)
+      offers a decent list of categories for unacceptable types of content. For
+      example, their
+      [thumbnail policy](https://support.google.com/youtube/answer/9229980?hl=en&ref_topic=9282679#zippy=%2Cage-restricted-thumbnails-and-thumbnail-removal)
+      includes some information on types of sexual and violent content that
+      either can't be used or can be used behind a flag comparable to ours.
+      - Additionally, see the [#this-will-not-be-perfect](#this-wont-be-perfect)
+        section for additional
+
+## Implementation Plan
+
+1. Ping a WordPress GitHub account maintainer to request that they create the
+   `WordPress/openverse-sensitive-terms` repository.
+2. Initialize the repository with a `README.md` and a blank
+   `sensitive-terms.txt` file.
+3. Create a PR which initializes the sensitive terms list by copying all terms
+   in the "List of Dirty, Naughty, Obscene, and Otherwise Bad Words" repository.
+   _I am not including the link here to prevent accidental viewing of the
+   repository, but feel free to find it within the GitHub organization
+   "LDNOOBW"._
+4. Identify willing reviewers from the `@WordPress/openverse-maintainers` group
+   who are comfortable with looking at the list and reviewing the PR, looking
+   for terms which should be removed or missing terms which should be added.
+5. Merge the PR, and now the list is available for use in Openverse projects.
+6. Publish a "Make" post about the creation of the list and its purpose.
+
+## This Will Not Be Perfect
+
+By default, we accept that this list and its usage will not be perfect. At times
+we may accidentally include terms which inadvertently cause harm to groups or
+individuals. We may also accidentally expose users to undesired content in the
+implementation of this list. Despite the potential for these transgressions this
+work will always be based on the following assumptions:
+
+1. Safety first. Given the choice, we would prefer to incorrectly restrict
+   non-sensitive content rather than accidentally show sensitive content.
+2. Avoid censorship. This list will not be used to block or deny access to
+   content, rather to make the viewing of content opt-in. The Openverse project
+   will use other means to explicitly block or remove content. Removing
```

Comment:
Is it worth clarifying that we would only block or remove content "in line with our content policy"? I assume that's following some WordPress Foundation content policy, but might be distinct, not sure.

Reply:
I'm not sure either. Leaving it as ambiguous as possible (the way it is now) at least prevents us from saying something strictly incorrect. I'd like if we could ask someone familiar with WP Foundation's policies on this. Specifically, how would they like us to communicate the fact that we do reserve the right to remove content from Openverse for reasons other than DMCA and broad "illegality". (I totally agree that we need better understand how to communicate _that_ aspect as well.)

Reply:
Agreed that for now, we'll leave this intentionally ambiguous.